### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by duweihua** [aesrgan/A-ESRGAN#2]
Does the genarator and discriminator's function of public code is the really implementation  in paper ?

**Comment by aesrgan** [comment]
Yes they are. Have you got any specific questions?

**Comment by duweihua** [comment]
>
Pixel-wise relativistic loss of D and G in the paper, but I do not find and understand the pixel-wise relativity in the code, it's not the standard GAN loss?
![image](https://user-images.githubusercontent.com/43801202/148639007-7deccbdd-1939-4656-af5b-c54bfd624e34.png)
![image](https://user-images.githubusercontent.com/43801202/148639015-61bfeb89-916c-421d-b617-acd339837eea.png)

**Comment by aesrgan** [comment]
Yes it's not the standard gan loss .If you look at the cri_gan implementation in the basicsr library, you can find that it is implemented using BCEWITHLOGITSLOSS in torch.nn, which is a BCE loss with sigmoid function.

**Comment by duweihua** [comment]
> 

I know the implementation in basicsr library, but I can ubderstand the pixel-wise relativity in the public code of your implementation, where is the relativity?

**Comment by zeratoo70** [comment]
What is meaning of second argument (True) in cri_gan loss? Where it is used why it has this or that value?
